### ISSUE TITLE
Different mean IOU on validation set

### ISSUE THREAD

**Issue by stephenjia** [DrSleep/tensorflow-deeplab-lfov#14]
Hi,
Thanks a lot for open-source this code. It is very helpful. However, I run into a problem when running the evaluation code on validation set. I did not change anything but just load the model.ckpt-pretrained. But surprisingly, it gets 70.4 instead of 67 as mentioned in the README. Can I ask you what could be the reason?
Thanks a lot!

**Comment by xtudbxk** [comment]
I just done some experiments, and fount that the "model.ckpt-init" is same with  "init.caffemodel" in deeplab website, and the "model.ckpt-pretrained" is the same with "train2_iter_6000.caffemodel". So May be the 67 miou is trained on "model.ckpt-init", not the "model.ckpt-pretrained".

**Comment by stephenjia** [comment]
Thanks for your reply. But if "model.ckpt-init" is same with "init.caffemodel", then the classifier params are randomly initialized and I do not think that will bring the number of 67.
We switch to author's repository with resnet, that is, deeplab v2 implementation and that code works very well.
